script/scp-hosts.sh

```diff
@@ -28,6 +28,7 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 fi
 
 workersGroup=(${workers//,/ })
+declare -A workersGroupMap
```

Comment:
```declare -A``` is not working in ```mac``` system.

I think a new implementation is needed.

Reply:
Hi @SbloodyS ,
Bash version on Mac is 3.x while on Linux its verion is usually 4.x +. Bash 3.x does not support `declare -A`.

If we have to support Bash 3.x, we need to avoid declaring a variable as map.

Reply:
We can replace the map with two arrays, with assigning the key value pair the same index. However it seems that the worker groups are not needed in those scripts. Can I just remove worker groups and add them back if they were required in future versions?

Reply:
Sorry for late reply. Yes. You can remove worker groups since it is not currently used. @paul8263